WordPress/twentytwentyfive
# Try: Use copyright in footer instead of `Twenty Twenty-Five` string.

**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/385

- Use the copyright pattern instead of hardcoding `Twenty Twenty-Five`
- Remove the text color from the pattern.
- Wrap the pattern in a group with text color for the "Portfolio Homepage" to match the designs.

**Screenshots**

https://github.com/user-attachments/assets/d2d832f4-29c5-43a6-8b62-4d1e85ca5bff


**Testing Instructions**

Copyright in footer

1. On the site editor, edit the footer.
2. Confirm that the three footers described in the issue are using the copyright pattern instead of the `Twenty Twenty-Five` hardcoded string.

Copyright in "Portfolio Homepage"

1. Create a page.
2. Use the "Portfolio Homepage"
3. Confirm the copyright pattern is still using the same text color.
